conf/modules/collective_tracking_control.xml

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="collective_tracking_control" dir="ctc">
```

Comment:
In order to limit the number of directory in the modules folder, would it be possible to move this to either `multi` or `ctrl` dir ? Same for the target part (and DCF, but that is for another PR ;) )

Reply:
Sure, I agree.